### PR TITLE
Fixed more typos

### PR DESCRIPTION
**Pull request by tony-xia**

### CODE REVIEW

File: packages/experiments/src/components/TilesList/TilesList.tsx [hunk 1]
```diff
@@ -543,7 +543,7 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
 
     if (pageSpecificationCache.byIndex[startIndex]) {
       // If the page specification has already been calculated, return it.
-      // List recalculates all pages if any input changes, so this memoization
+      // List recalculates all pages if any input changes, so this memorization
```
Comment (ecraig12345) [inline packages/experiments/src/components/TilesList/TilesList.tsx:546]: This was correct previously. https://en.wikipedia.org/wiki/Memoization
```suggestion
      // List recalculates all pages if any input changes, so this memoization
```

**Comment by tony-xia** [comment]
thanks @ecraig12345 for your feedback. I updated them accordingly

**Comment by ecraig12345** [comment]
Thanks for the updates but this is still missing a change file. Basically clone the repo and run `yarn` then `yarn change`, or see our [contributing docs](https://github.com/microsoft/fluentui/wiki/Contributing) for more details.

**Comment by tony-xia** [comment]
Sorry, i might be wrong, i reverted unnecessary changes. there is only 2 typo fixes at the moment. https://github.com/microsoft/fluentui/pull/13247/files

**Comment by ecraig12345** [comment]
The changes are correct now, but you need an additional file called a change file which is generated using the steps I described.

**Comment by tony-xia** [comment]
Updated. Thanks @ecraig12345 for your feedback

**Pull request merged**